pandas/tests/test_categorical.py

```diff
@@ -57,6 +57,12 @@ def test_getitem_listlike(self):
         expected = c[np.array([100000]).astype(np.int64)].codes
         tm.assert_numpy_array_equal(result, expected)
 
+    def test_getname_category(self):
```

Comment:
parametrize with all accessor methods

Reply:
you can use a lambda expression

```
@pytest.mark.parametrize("method",
[
   lambda x: x.cat.set_categories([1, 2,3 ]),
   lambda x: x.cat.reorder_categories([2,3, 1], ordered=True),
])
def test_getname_categorical_accessor(self, method):
     s = ....
    expected = 'A'
    result = method(s).name
    assert result == expected
```

Reply:
The current test function is also fine only right? What is the difference? Performance?